packages/api/src/routes/node.ts

```diff
@@ -52,9 +52,9 @@ export type FilterGetPeers = {
 
 export type SyncingStatus = {
   /** Head slot node is trying to reach */
-  headSlot: Slot;
+  headSlot: string;
```

Comment:
After merging ssz refactor this should not be necessary since uint in JSON form is a string